sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeSentimentTests.cs

```diff
@@ -455,6 +455,7 @@ private void CheckAnalyzeSentimentProperties(DocumentSentiment doc, bool opinion
                         Assert.AreEqual(0, minedOpinions.Aspect.ConfidenceScores.Neutral);
                         Assert.IsTrue(CheckTotalConfidenceScoreValue(minedOpinions.Aspect.ConfidenceScores));
                         Assert.IsNotNull(minedOpinions.Aspect.Offset);
+                        Assert.IsNotNull(minedOpinions.Aspect.Length);
```

Comment:
Analyze sentiment tests need some love. Issue for that is here => https://github.com/Azure/azure-sdk-for-net/issues/14833